src/PublicApi/CatalogItemEndpoints/Create.cs

```diff
@@ -45,12 +43,15 @@ public override async Task<ActionResult<CreateCatalogItemResponse>> HandleAsync(
 
             if (newItem.Id != 0)
             {
-                var picName = $"{newItem.Id}{Path.GetExtension(request.PictureName)}";
-                if (await _webFileSystem.SavePicture(picName, request.PictureBase64, cancellationToken))
-                {
-                    newItem.UpdatePictureUri(picName);
-                    await _itemRepository.UpdateAsync(newItem, cancellationToken);
-                }
+                // At this point time, the Admin application uses the default catalog item image for any new product item.
```

Comment:
How about using a code comment like this:\
```
We disabled the upload functionality and added a default/placeholder image to this sample due to a potential security risk pointed out by the community. More info in this issue: https://github.com/dotnet-architecture/eShopOnWeb/issues/537 In production, we recommend using blob storage and deliver the image via CDN.
````

Reply:
Yes, this sounds much better. Will modify the comment.